app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :upvoted_users, through: :votes, source: :user
+
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: { scope: :category }
+  validates :creator, presence: true
+  validates :publication_year, presence: true
+
+  def self.order_albums
+    albums = Work.where(category: 'album')
+    return albums.sort_by{|album| album.votes.count}.reverse
```

Comment:
You have almost exactly the same code for these 3 methods. Could you DRY them up somehow, perhaps by passing in the category as a parameter?